test/layouts/Layout.test.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { describe, expect, it, vi } from 'vitest';
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import * as hooks from '@hooks';
+import { SessionContext } from '@contexts';
+import { ThemeProvider } from '@mui/material/styles';
+import { NavBar } from '@components/NavBar';
+import Layout from '../../src/layouts/Layout';
+import theme from '../../src/theme';
+import createMatchMedia from '../helpers/createMatchMedia';
+
+const MockLayout = () => (
+  <ThemeProvider theme={theme}>
+    <BrowserRouter>
+      <Layout ariaLabel="test">
+        <NavBar />
+      </Layout>
+    </BrowserRouter>
+  </ThemeProvider>
+);
+
+describe('Desktop view', () => {
+  it('Renders breadcrumb when logged in, layout grid-template-rows as 64px 64px 1fr 280px', () => {
+    vi.spyOn(hooks, 'useNotification').mockReturnValue({
```

Comment:
Slightly off topic of the PR. I checked the `hook/useNotification.js` file and noticed that it was importing the context file and the call `useContext(NotificationContext)` was in the hooks folder. This is the pattern we should all be using for useContext() hooks?

Reply:
Good catch. No, I think we should either just have `Notification` use straight up `useContext` or migrate everything over to the custom hook. I'll open an issue for it.

Reply:
Hm...scratch that. It might be reasonable to format it like what we have now to `useNotification` and `NotificationContext` since `Notification` is one of our global states used across multiple files. I think that's what we're doing with `useSession` and `SessionContext` is doing.

Reply:
Sounds good to me. Thanks for sharing. I've also come across the use of custom error messages declared in Context files which seemed kinda nice.
 
Example:
```
const FooContext = createContext();

export const FooProvider = ({ children }) => {
  //... 
  return <FooContext.Provider value={value}>{children}</FooContext.Provider>;
};

export const useFooContext = () => {
  const context = useContext(FooContext);
  if (!context) {
    throw new Error('useFooContext must be used within an FooProvider');
  }
  return context;
};
```